networking_oneview/ml2/drivers/oneview/synchronization.py

```diff
@@ -55,9 +55,11 @@ def get_session(self):
     def synchronize(self):
         self.delete_outdated_flat_mapped_networks()
         self.create_oneview_networks_from_neutron()
-        self.delete_unmapped_oneview_networks()
-        self.synchronize_uplinkset_from_mapped_networks()
-        self.recreate_connection()
+
+        if common.CONF.oneview.force_sync_delete_ops:
+            self.delete_unmapped_oneview_networks()
+            self.synchronize_uplinkset_from_mapped_networks()
+            self.recreate_connection()
```

Comment:
It would be good to have a `else` statement for a LOG.warning saying deletion is not enabled.

Reply:
Yes @hugonicodemos

Reply:
Apparently we have a DEBUG saying the sync started... should we have something here too?